src/requirements/admin/migrate-engineering-liberal-arts-data.ts

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable no-console */
+
+import { usernameCollection } from '../../firebase-admin-config';
+import { requirementAllowDoubleCounting } from '../requirement-frontend-utils';
+import getUserRequirementDataOnAdmin from './requirement-graph-admin-utils';
+
+const REQUIREMENT_ID = 'College-EN-Liberal Studies: 6 courses';
+
+async function runOnUser(userEmail: string) {
+  const {
+    selectableRequirementChoices,
+    onboardingData,
+    userRequirements,
+    requirementFulfillmentGraph,
+  } = await getUserRequirementDataOnAdmin(userEmail);
+  const userRequirementMap = Object.fromEntries(userRequirements.map(it => [it.id, it]));
+
+  const connectedCourses = requirementFulfillmentGraph.getConnectedCoursesFromRequirement(
+    REQUIREMENT_ID
+  );
+  const coursesNotConnectedToOtherRequirementThatDoesNotAllowDoubleCounting = connectedCourses.filter(
```

Comment:
Would a better name for this be `connectedCoursesOnlyConnectedToDoubleCountedReqs`?